packages/medusa/src/api/routes/admin/tax-rates/update-tax-rate.ts

```diff
@@ -40,6 +40,9 @@ import { isDefined } from "medusa-core-utils"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesTaxRateReq"
+ * x-codegen:
+ *   method: update
+ *   queryParams: AdminPostTaxRatesTaxRateParams
```

Comment:
Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L68-L73